README.md

```diff
@@ -60,100 +60,105 @@ billing:
-        mandatory: false
-        validator:
-      - name: number_test
-        description: Validator range for value of dataType number.
+      - name: k8s_node_count
+        description: The node except master count of K8S cluster.
         kind: variable
         type: number
         mandatory: true
-        #minimum=10|maximum=100: The age must be greater than or equal to 10 and less than or equal to 100.
-        validator: minimum=10|maximum=100
-      - name: string_test
-        description: Validator length for value of dataType string.
-        kind: variable
-        type: string
-        mandatory: true
-        # minLength=6|maxLength=16: The value must be at least 6 characters long and no more than 16 characters long.
-        validator: minLength=6|maxLength=16
-      - name: enum_test
-        description: Validator enum for value of dataType string.
+        # Validators for value of dataType number.
+        ## minimum=2|maximum=10: The value must be greater than or equal to 2 and less than or  equal to 10.
+        validator: minimum=2|maximum=10
+      - name: admin_passwd
+        description: The password of machine of K8S cluster.
         kind: variable
         type: string
         mandatory: true
-        # enum=["red","yellow","green"]: the value must be in the list of enum. 
-        validator: enum=["red","yellow","green"]
-      - name: pattern_test
-        description: Validator pattern for value of dataType string.
-        kind: variable
-        type: string
-        mandatory: false
-        # pattern=*e*: the value matches any string that contains the specified. 
-        validator: pattern=*e*
+         # Validators for value of dataType string.
+         ## minLength=8|maxLength=16: The value must be at least 8 characters long and no more than 16 characters long.
+         ## pattern=^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,16}$: the value matches any string that contains the specified. 
+        validator: minLength=8|maxLength=16|pattern=^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,16}$
    deployer: |
-      variable "secgroup_id" {}
-      data "huaweicloud_availability_zones" "myaz" {}
-      data "huaweicloud_compute_flavors" "myflavor" {
-          availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
-          performance_type  = "normal"
-          cpu_core_count    = 2
-          memory_size       = 4
+      data "huaweicloud_availability_zones" "osc-az" {}
+
+      variable "admin_passwd" {
+      type = string
+      description = "The root password for all nodes of the K8S cluster."
+      }
+
+      variable "k8s_node_count" {
+      type = number
+      description = "The count of nodes except master of the K8S cluster."
+      }
+
+      resource "huaweicloud_compute_instance" "k8s-master" {
+      name = "k8s-master"
+      availability_zone = data.huaweicloud_availability_zones.osc-az.names[0]
+      flavor_id = "c7.xlarge.4"
```

Comment:
better to make this c7.xlarge.4 configurable, mybe with different flavors type like performance、normal...

Reply:
Ok. I will define a variable flavor_id and use it here

Reply:
no `flavor_id` please, the deployment already be able to pass the flavor name to the deploy process yet.

Reply:
I define the performance and normal flavor in flavors. The property of flavor contains worker_nodes_count, flavor_id, master_nodes_count.